deploy/helm/hdfs-operator/crds/crds.yaml

```diff
@@ -34,6 +34,40 @@ spec:
                       minimum: 0.0
                       nullable: true
                       type: integer
+                    kerberos:
+                      description: Configuration to set up a cluster secured using Kerberos.
+                      nullable: true
+                      properties:
+                        kerberosSecretClass:
+                          default: kerberos
+                          description: Name of the SecretClass providing the keytab for the HDFS services.
+                          type: string
+                        requestNodePrincipals:
+                          default: false
+                          description: Wether a principal including the Kubernetes node name should be requested. The principal could e.g. be `HTTP/my-k8s-worker-0.mycorp.lan`. This feature is disabled by default, as the resulting principals can already by existent e.g. in Active Directory which can cause problems.
+                          type: boolean
+                        tlsSecretClass:
```

Comment:
So you suggest something like the following?
```
      spec:
        clusterConfig:
          security:
            kerberos:
              tlsSecretClass: tls
              kerberosSecretClass: kerberos-$NAMESPACE
              wireEncryption: {{ test_scenario['values']['wire-encryption'] }}
```
I would say all these settings are security-related, yes. But for other product we have spec.clusterconfig.{authentication,authorization}, without putting it below spec.clusterconfig.security. So I guess the current way is more konsitent, WDYT?

Reply:
More like:

```yaml
      spec:
        clusterConfig:
          security:
            tlsSecretClass: tls
            kerberos:
              kerberosSecretClass: kerberos-$NAMESPACE
            wireEncryption: {{ test_scenario['values']['wire-encryption'] }}
```

Since TLS (for example) doesn't have much to do with Kerberos compared to any other hypothetical authn method. Then again, there's the question of how likely that is to happen anyway...

Reply:
Implemented in 4a2afb92659f448ee25e2290459a67e73afebe6c

Reply:
We have decided on new structure in Arch meeting